legacy/expressions/migrate_test.go

```diff
@@ -300,35 +344,40 @@ func TestLegacyTests(t *testing.T) {
 	require.NoError(t, err)
 
 	var tests []legacyTest
-	err = json.Unmarshal(legacyTestData, &tests)
+	d := json.NewDecoder(bytes.NewReader(legacyTestData))
+	d.UseNumber()
```

Comment:
means numbers are parsed as strings -by default it was using `float64` but that was causing failing tests like 6.23 != 6.299999....